lib/video_info/providers/vimeo.rb

```diff
@@ -26,17 +50,34 @@ def embed_url
       end
 
       def date
-        Time.parse(_video['upload_date'], Time.now.utc).utc
+        Time.parse(_video['created_time'], Time.now.utc).utc
       end
 
       def view_count
-        _video['stats_number_of_plays'].to_i
+        _video['stats']["plays"].to_i
       end
 
       private
 
+      def _clean_options(options)
+        headers = [super, _authorization_headers, _api_version_headers]
+        headers.inject(&:merge)
+      end
+
+      def _api_version
+        "3.2"
+      end
+
+      def _authorization_headers
+        { "Authorization" => "bearer #{api_key}" }
+      end
+
+      def _api_version_headers
+        { "Accept" => "application/vnd.vimeo.*+json;version=#{_api_version}"}
```

Comment:
Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.